pint_pandas/pint_array.py

```diff
@@ -309,12 +311,21 @@ def __setitem__(self, key, value):
             # doing nothing here seems to be ok
             return
 
+        try:
+            next(i for i in self._data if pd.notna(i))
```

Comment:
What's this needed for?

Reply:
That was some vestigial code that needs to be further deleted.